core/domain/topic_services.py

```diff
@@ -246,6 +284,36 @@ def save_new_topic(committer_id, topic):
         })])
 
 
+def update_subtopic_page(topic_id, subtopic_id, change_list):
+    """Applies a changelist and updates the subtopic page model entry in the
+    datastore.
+
+    Args:
+        topic_id: str. The topic id in which the subtopic is present.
+        subtopic_id: int. The id of the subtopic.
+        change_list: list(TopiChange). The list of changes that are to be
+            applied to the subtopic page object.
+    """
+    subtopic_page_id = topic_domain.SubtopicPage.get_subtopic_page_id(
+        topic_id, subtopic_id)
+    subtopic_page = get_subtopic_page_by_id(subtopic_page_id)
+    try:
+        for change in change_list:
+            if change.cmd == topic_domain.CMD_UPDATE_SUBTOPIC_PAGE_PROPERTY:
```

Comment:
There should be an "else raise Error" at the end of this chain. Otherwise it's not clear that this is exhaustive.

(Please make sure this is the case in other similar functions too.)

Reply:
Done.